packages/roosterjs-editor-api/lib/utils/processList.ts

```diff
@@ -64,7 +66,10 @@ function applyListFormat(node: Node, formats: string[], isDarkMode: boolean) {
         {
             fontFamily: formats[0],

```

Comment:
Looking at getComputedStyles and this code block, it seems like we're relying on hard coded positioning of returned attributes. We should likely refactor this to return a typed object that we can then reference, as opposed to this hacky workaround. Thoughts on undertaking this work now?

Reply:
Sounds like an improvement to me. But if it's too high cost, I'm fine with leaving as is for now and logging a Wi to return to later.

Reply:
I'd rather do this in the main branch and port over. Don't want this fix to live only in dark mode.

Reply:
Yeah, and it's complicated to do... will defer this.